src/matrices.jl

```diff
@@ -209,14 +213,14 @@ length, the recurrences are only calculated until the length of the shortest one
 See [`RecurrenceMatrix`](@ref) for details, references and keywords.
 See also: [`CrossRecurrenceMatrix`](@ref).
 """
-function JointRecurrenceMatrix(x, y, ε; kwargs...)
+function JointRecurrenceMatrix(x, y, ε; parallel::Bool = false, kwargs...)
     n = min(size(x,1), size(y,1))
     if n == size(x,1) && n == size(y,1)
-        rm1 = RecurrenceMatrix(x, ε; kwargs...)
-        rm2 = RecurrenceMatrix(y, ε; kwargs...)
+        rm1 = RecurrenceMatrix(x, ε; parallel = parallel, kwargs...)
```

Comment:
Since `parallel` is used everywhere anyway, should you just propagate it along with the rest of `kwargs...` ?

Reply:
Good point - will do!